diff --git a/node_modules/@clerk/expo/dist/specs/NativeClerkModule.android.js b/node_modules/@clerk/expo/dist/specs/NativeClerkModule.android.js
index deba0f1..4016857 100644
--- a/node_modules/@clerk/expo/dist/specs/NativeClerkModule.android.js
+++ b/node_modules/@clerk/expo/dist/specs/NativeClerkModule.android.js
@@ -1,7 +1,13 @@
 let expo = require("expo");
 
 //#region src/specs/NativeClerkModule.android.ts
-var NativeClerkModule_android_default = (0, expo.requireNativeModule)("ClerkExpo");
+var NativeClerkModule_android_default;
+try {
+  NativeClerkModule_android_default = (0, expo.requireNativeModule)("ClerkExpo");
+} catch (e) {
+  console.error("Failed to load NativeClerkModule ClerkExpo:", e);
+  NativeClerkModule_android_default = null;
+}
 
 //#endregion
 module.exports = NativeClerkModule_android_default;
